File: src/test/java/br/com/rcc_dev/testes/startup/GuiceModuleMock.java
package br.com.rcc_dev.testes.startup;

import br.com.rcc_dev.testes.controls.CepControl;
import br.com.rcc_dev.testes.mocks.CepControlMock;
import com.google.inject.AbstractModule;

public class GuiceModuleMock extends AbstractModule {
  
  @Override
  protected void configure(){
    bind(CepControl.class).to(CepControlMock.class);
  }
  
}

File: src/test/kotlin/br/com/rcc_dev/testes/startup/GuiceModuleMock.kt
package br.com.rcc_dev.testes.startup

import br.com.rcc_dev.testes.controls.CepControl
import br.com.rcc_dev.testes.mocks.CepControlMock
import com.google.inject.AbstractModule

class GuiceModuleMock: AbstractModule() {

  override fun configure() {
    super.configure()
    bind(CepControl::class.java).to(CepControlMock::class.java)
  }

}